README.md

```diff
@@ -14,7 +14,7 @@
 
 ## Getting Started
 
-**This a set up of my mobile version of my portfolio project. It adds the works and about sections to the already created headline section. It uses a figma template 2 from microverse student portal..**
+**This a set up of my mobile version of my portfolio project. It adds a contact form to the already created headline, works and about sections. It uses a figma template 2 from microverse student portal..**
```

Comment:
- [ **OPTIONAL** ] What do you think if you put the project's description just after the project's title by starting with a title **Description** then details below the title as it is in the Microverse [README template](https://github.com/microverseinc/readme-template/blob/master/README.md). And if possible fulfill the **Getting Started** section with some instructions on how to set up your project locally. You can take a look at this [README](https://github.com/tresorsawasawa/MyPortfolio/edit/develop/README.md) as example.